Source/WebGPU/WebGPU/Device.mm

```diff
@@ -961,10 +967,10 @@ static void setOwnerWithIdentity(id<MTLResourceSPI> resource, auto webProcessID)
         }
     }
 
-    [[vertex]] void vsUshort(device const IndexDataUshort* indexData [[buffer(0)]],
+    [[vertex]] void vsUshortICB(device const IndexDataUshort* indexData [[buffer(0)]],
         device ICBContainer *icb_container [[buffer(1)]], // <-- must match Device::bufferIndexForICBContainer()
```

Comment:
Nit, not for this patch: since these are string literals, you could #define the bufferIndexForICBContainer value to re-use in the shader here, instead of relying on the comments.

Reply:
oh that is a good idea